napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -378,11 +377,17 @@ def __init__(
             self.action_button.setText(trans._("install"))
             self.action_button.setObjectName("install_button")
 
-    def _handle_npe2_plugin(self):
+    def _handle_npe2_plugin(self, npe_version):
+        if npe_version == 1:
```

Comment:
Looks like `npe_version` is sometimes an `int` and sometimes a `str`. Optional, but is it worth considering an enum here? Or do you mostly want this to be an `int` and the `str` is a special case that we can just live with for now?